Jenkinsfile

```diff
@@ -16,7 +16,10 @@ pipeline {
 		stage('install') {
 			steps {
 				nvm(getNodejsVersion()) {
-					sh 'npm i --registry https://npm.lisk.com'
+					sh '''
```

Comment:
I am not sure if this change is required, maybe we can update the lock file properly?